python/lsst/daf/butler/_butler.py

```diff
@@ -348,7 +348,7 @@ def makeRepo(
 
         Parameters
         ----------
-        root : `str` or `ButlerURI`
+        root : `str` or `lsst.resources.ResourcePath`
```

Comment:
Could probably be `pathlib.Path`, too.

Reply:
I am wondering whether `ResourcePathExpression` works in Sphinx. I suppose I should try it.

Reply:
Ironically the type annotation for makeRepo only says `str` so that also needs fixing.